plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -204,7 +204,7 @@
     },
     "bls12_381_G2_neg": {
         "cpu": {
-            "arguments": 307813,
+            "arguments": 129165,
```

Comment:
That's quite a reduction.

Reply:
That's unrelated to this PR.  I updated the benchmarking results for the bls12-381 functions at some point and they've only just made it into `builtinCostModel.json`.  In retrospect I should probably have updated those in a separate PR to avoid confusion.